Samples/iOS-Swift/iOS-Swift/AppDelegate.swift

```diff
@@ -27,9 +27,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             options.enableCoreDataTracking = true
             options.enableProfiling = true
             options.attachScreenshot = true
+            options.anrEnable = true
+            options.anrTimeoutInterval = 2 
```

Comment:
maybe align with java - it's 5 seconds at the moment: https://github.com/getsentry/sentry-java/blob/840fc7ea60762f0f532be189f899f628f3319e90/sentry-android-core/src/main/java/io/sentry/android/core/SentryAndroidOptions.java#L20

Reply:
I believe 5 seconds to be too much to report an ANR. 2 seconds of the app being unresponsive should already be addressed be the developer IMO.